.travis.yml

```diff
@@ -21,6 +24,9 @@ before_script:
 script:
   - npm test
   # Only run funcs for LTS releases
+  - if [ "$TRAVIS_NODE_VERSION" = "11" ]; then npm run functionaltest; fi
```

Comment:
Nit: 11 isn't an LTS release. It's okay if we want to establish a new criteria for what belongs here, but the comment should be updated

Reply:
updated to run func tests on current release as well